core/scale/types.hpp

```diff
@@ -21,8 +21,11 @@ namespace kagome::scale {
    */
   using CompactInteger = boost::multiprecision::cpp_int;
 
-  /// @brief OptionalBool is internal extended bool type
+  /**
+   * @brief OptionalBool is internal extended bool type
+   */
```

Comment:
Why? It the same

Reply:
Why change the comment? Because others in this file are commented this way.